nomad/structs/structs.go

```diff
@@ -2897,6 +2897,15 @@ func (r *ReschedulePolicy) Validate() error {
 	}
 	var mErr multierror.Error
 	// Check for ambiguous/confusing settings
+	if r.Attempts > 0 {
+		if r.Interval <= 0 {
+			multierror.Append(&mErr, fmt.Errorf("Interval must be a non zero value if Attempts > 0"))
+		}
+		if r.Unlimited {
+			multierror.Append(&mErr, fmt.Errorf("Reschedule Policy with Attempts = %v, Interval = %v, "+
+				"and Unlimited = %v is ambiguous", r.Attempts, r.Interval, r.Unlimited))
```

Comment:
What does ambiguous mean in this case? Clarify and maybe offer a hint of what the operator should change for this to be successful (if it fits into a reasonably-sized error message).

Reply:
If attempts are > 0 then you are limiting by attempts in an interval, so you can't also have unlimited=true or unlimited attempts. will try to expand this error message but I can't make it too verbose either